package.json

```diff
@@ -14,17 +14,18 @@
 	"main": "./dist/index.js",
 	"module": "./dist/index.mjs",
 	"types": "./dist/index.d.ts",
-	"files": [
-		"dist"
-	],
+	"bin": {
+		"browserslist-to-es-version": "./dist/cli.js"
+	},
+	"files": ["dist"],
 	"scripts": {
-		"build": "rslib build",
+		"build": "rslib build && npx tsc src/cli.ts --outDir dist --target es2021 --module es2022 --moduleResolution node --allowSyntheticDefaultImports",
```

Comment:
Can we use Rslib to build `src/cli.ts`? See https://rslib.rs/config/lib/bundle#set-entry

Reply:
Sure thing. I've updated this branch.